src/libs/ReportActionsUtils.ts

```diff
@@ -117,6 +130,17 @@ function isWhisperAction(reportAction: OnyxEntry<ReportAction>): boolean {
     return (reportAction?.whisperedToAccountIDs ?? []).length > 0;
 }
 
+function isWhisperActionTargetedToOthers(reportAction: OnyxEntry<ReportAction>): boolean {
+    if (!isWhisperAction(reportAction)) {
+        return false;
+    }
+    if (!currentUserAccountID) {
+        return true;
+    }
+    const whisperedToAccountIDs = reportAction?.whisperedToAccountIDs ?? [];
+    return !whisperedToAccountIDs.includes(currentUserAccountID);
```

Comment:
Can't you simply this like so?

```suggestion
    return !reportAction?.whisperedToAccountIDs?.includes(currentUserAccountID);
```